src/app/auth/services/auth.service.ts

```diff
@@ -0,0 +1,71 @@
+import { Inject, Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { IUser } from '../models/user.interface';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../../environment/environment';
+import { constants } from '../../core/constants/app-constants';
+import { DOCUMENT } from '@angular/common';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  storageSub: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(
```

Comment:
would be nice to rename it to Subj at the end, Sub kinda feels like its a subscription (small thing, but I tohught it was a subscription at first :D)